src/router/index.tsx

```diff
@@ -15,6 +16,9 @@ export default function Routes() {
         <Route exact path="/">
           <Home />
         </Route>
+        <Route exact path="/about-us">
```

Comment:
```suggestion
        <Route path="/about-us">
```

I don't think this Route needs to have `exact` property.  
Ref: https://reactrouter.com/web/api/Route

Reply:
To be clear, why we only put `exact` only for path `/`.